libs/domain/search/src/services/resolvers/category-page-title-meta.resolver.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  ElementResolver,
+  PageMetaResolver,
+  ResolverScore,
+} from '@spryker-oryx/core';
+import { inject } from '@spryker-oryx/di';
+import { FacetValue } from '@spryker-oryx/product';
+import { RouterService } from '@spryker-oryx/router';
+import { map, Observable, of, switchMap } from 'rxjs';
+import { FacetListService } from '../facet-list.service';
+
+export class CategoryPageTitleMetaResolver implements PageMetaResolver {
+  constructor(
+    protected router = inject(RouterService),
+    protected facets = inject(FacetListService)
+  ) {}
+
+  getScore(): Observable<number> {
+    return this.router
+      .currentQuery()
+      .pipe(
+        map((query) =>
+          query?.category
+            ? ResolverScore.OverrideDefault
+            : ResolverScore.NotUsed
+        )
+      );
+  }
```

Comment:
what about we have a base score (say 0), and then we get additional points. In this example it would add 1 point since it's a category. 

Say we have a special resolver for Canon categories, we'd get another point. 
Say we have a special resolver Canon in week 41, we'd get another point.

Reply:
Unfortunately we can't use here reliable calculation between resolvers as they are separate

Reply:
Correct. If we have other teams inside Spryker creating resolvers (say PDP for marketplace), as well as partners add resolvers as well as customers write theirs. We cannot avoid conflicts.